chrome-options.js

```diff
@@ -246,6 +250,10 @@
       hashPosition++;
     }
 
+    if (option.type == 'html') {
+      return addHtml(option);
+    }
```

Comment:
I'd rather put this next to the [layout options](https://github.com/fent/chrome-options#layout-fields)

Either after `case 'row':` below, or after `case 'h3':` in `addTabOptions()`